### PR TITLE
Remove old INSTALL.msg

### PR DESCRIPTION
**Pull request by ericonr**
I didn't revbump any package because this is a change that I feel doesn't warrant a rebuild.

### CODE REVIEW

**Comment by travankor** [comment]
`dnscrypt-proxy` and `yq-go` can also be removed

**Comment by ericonr** [comment]
@travankor thanks! Added all of them.

Regarding the lightdm-webkit-greeter package, its replacement was added in 2018 (73875399), so it should be ok to remove the whole subpackage.

**Comment by ericonr** [comment]
@CameronNemo the packages you mentioned are here

**Comment by ericonr** [comment]
If removing the lightdm-webkit2-greeter package is bad, lmk and I will only remove the INSTALL.msg part.

File: srcpkgs/neomutt/INSTALL.msg [hunk 1]
```diff
@@ -1,2 +0,0 @@
-As of 20171013, neomutt is no longer mutt. System config is in /etc/neomuttrc
-Due to upstream's choice, we have removed the alternatives for neomutt files.
```
Comment (Vaelatern) [inline srcpkgs/neomutt/INSTALL.msg:2]: Goodness, didn't realize this was so long ago.

**Comment by Vaelatern** [comment]
I'd rather only remove INSTALL.msgs in this.

**Comment by ericonr** [comment]
@Vaelatern fixed!

**Pull request merged**